docs/learn/smart-contract-developers/edit-token-metadata.md

```diff
@@ -0,0 +1,137 @@
+---
+sidebar_label: '💽 Edit Token Metadata'
+sidebar_position: 3
+description: Learn how to update LSP4 Metadata of digital assets on LUKSO.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Edit Token Metadata
+
+In this guide, you will learn how to edit the [metadata](../../standards/tokens/LSP4-Digital-Asset-Metadata.md) of an [LSP7 Digital Asset](../../standards/tokens/LSP7-Digital-Asset.md) by updating its LSP2 data key. If you want to learn more about the contract deployment itself, please have a look at the [Getting Started](./getting-started.md) or [Create LSP7 Token](./create-lsp7-token.md) guides before you continue.
+
+To edit an LSP7 Digital Asset metadata, it is super easy, you will need to:
+
+1. get your assets ready (images, videos, etc.) and a metadata JSON file
+2. upload these files
+3. generate an encoded value
+4. write that value in the smart contract key-value store
+
+Let's go.
+
+:::tip Code repository
+
+You can find all the contracts, sample metadata, and scripts of the guide within our [`lukso-playground`](https://github.com/lukso-network/lukso-playground/tree/main/smart-contracts-hardhat) repository.
+
+:::
+
+## Get the encoded metadata URL
+
+For steps 1, 2 and 3, there is a [dedicated guide](../assets.md). Once your assets are uploaded, your metadata JSON file URL is ready and encoded, come back here.
```

Comment:
Maybe it's better to put this link in the pre-requisite above, or a specific section